docs/docker.rst

```diff
@@ -0,0 +1,30 @@
+How to run TRD in Docker
+========================
+
+It is possible to run TRD within Docker.
+
+You will need a pre-existing configuration file. It is recommended to run `configure.py` script outside of docker, then put the configuration yaml file in a `config` folder mounted in the container.
+
+The following mount points are expected:
+
+  * `config` folder containing config file
+  * `reports` folder containing payout reports
+
+Access to a signer endpoint and node endpoint are assumed in the host network.
+
+Here are the steps:
+
+1. Build the docker container
+  ::
+
+    docker build -t trd .
```

Comment:
@jdsika is it fine to use `trd` as a tag or should we stick with `tezos-reward-distributor:latest`? If we deploy it on dockerhub users should be able to pull by `docker pull tezos-reward-distributor/tezos-reward-distributor:latest` right?